### ISSUE TITLE
Build issues 

### ISSUE THREAD

**Issue by AbhishekChourasia16690** [John-Lluch/SWRevealViewController#679]
Hello,
I imported the SWRevealViewController.h and .m files in my project. Now unable to build my code since I have 20 build errors. Could you let me know the reason. I have attached the screenshot of it.

Thanks.
![error](https://cloud.githubusercontent.com/assets/22599405/22883871/3703de80-f218-11e6-98e0-3d3ae5cd9583.png)

**Comment by iDevelopper** [comment]
I think you perhaps inserted or deleted a character unfortunately in the code. Try to re-import the .h and .m files into your project.

**Comment by AbhishekChourasia16690** [comment]
Hello,
I tried re-importing the files, cleaned the project but I get the same error. 

Thanks

**Comment by iDevelopper** [comment]
Re-impoted from where? From a new download?

**Comment by AbhishekChourasia16690** [comment]
Yes from new download

**Comment by iDevelopper** [comment]
It is necessarily a problem of character somewhere:

http://stackoverflow.com/questions/8996811/expected-method-body-error

**Comment by AbhishekChourasia16690** [comment]
My code build successfully earlier, only after adding the SWRevealViewController.h and .m files in my project, I get the issue

**Comment by AbhishekChourasia16690** [comment]
Hello,
Am I missing anything else?

**Comment by iDevelopper** [comment]
No, I don't think. You could delete this library from your project and all code relative to it and see if you can build again.

**Comment by AbhishekChourasia16690** [comment]
Yes Im able to build when i removed the library. Also I have added the files manually in my project

**Comment by iDevelopper** [comment]
Can you share this project?

**Comment by AbhishekChourasia16690** [comment]
I wont be able to share the project due to authority issues. Please let me know if there is any other way

**Comment by iDevelopper** [comment]
I don't see. Try create a new simple sample project with this library. And you will see that all is all right.
